saleor/graphql/shop/types.py

```diff
@@ -128,6 +129,10 @@ class Shop(graphene.ObjectType):
                 "A language code to return the translation for."
             ),
         ),
+        in_shipping_zones=graphene.Boolean(
```

Comment:
Isn't it something that could be placed under `filter` field? At the first moment I wasn't sure what this field exactly does.

Reply:
I added better description. I hope it will be clear now